src/views/PaymentNetwork.vue

```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
   return `${BASE_CURRENCY.symbol}${Math.round(amount).toLocaleString()}`;
 }
 
-function formatCryptoCurrency(amount: number, currency): string {
-  return `${amount.toLocaleString()} ${currency.code}`;
-}
-
 watch(paymentTx, () => {
   modalPostPaymentOpen.value = !!paymentTx.value;
 });
 </script>
 
 <template>
-  <h1>Network plan</h1>
-  <p class="text-md mb-4 leading-5">Pay to support your network on Snapshot</p>
+  <h1 class="leading-[44px] mb-[2px]">Network plan</h1>
```

Comment:
```suggestion
  <h1 class="leading-[44px] mb-[2px]">Network plan</h1>
```
`text-2xl` should already be 44px line height and same font size as design

Reply:
`text-2xl` line-height is 50px